image/walker.go

```diff
@@ -92,11 +92,16 @@ func newPathWalker(root string) walker {
 
 func (w *pathWalker) walk(f walkFunc) error {
 	return filepath.Walk(w.root, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return errors.Wrap(err, "error walking path")
+		}
+		if info == nil {
```

Comment:
@runcom @stevvooe 
I tested `filepath.Walk` for any possible cases. I did not really find the result that error == nil and FileInfo == nil. 
Maybe `if info == nil` should be removed safely. Waiting for your proposal.